plugins/arDominionB5Plugin/js/imageflow.js

```diff
@@ -0,0 +1,35 @@
+(function ($) {
+  'use strict';
+  $(() => {
+    var node = $('div.atom-carousel');
+
+    if ($(node).length) {
+      $('.atom-slider-nav').slick({
+        slidesToShow: 3,
+        slidesToScroll: 1,
+        asNavFor: '.atom-slider-for',
+        dots: true,
+        centerMode: true,
+        instructionsText: $(node).data('carousel-instructions-text-image-link'),
+        regionLabel: 'image carousel',
+        variableWidth: true,
+        centerPadding: '60px',
+        nextArrow: '<button class="slick-next slick-arrow" type="button" style=""><span class="slick-next-icon" aria-hidden="true"></span><span class="slick-sr-only">' + $(node).data('carousel-next-arrow-button-text') + '</span></button>',
+        prevArrow: '<button class="slick-prev slick-arrow" type="button" style=""><span class="slick-prev-icon" aria-hidden="true"></span><span class="slick-sr-only">' + $(node).data('carousel-prev-arrow-button-text') + '</span></button>'
+      });
+
+      $('.atom-slider-for').slick({
+        centerMode: true,
+        slidesToShow: 1,
+        slidesToScroll: 1,
+        draggable: false,
+        swipe: false,
+        arrows: false,
+        fade: true,
+        asNavFor: '.atom-slider-nav',
```

Comment:
The line `asNavFor: '.atom-slider-nav',` seems to be needed otherwise the sliders are not associated correctly on initial load - the images start out not centered.  Leaving this line in place.